migration/src/androidTest/kotlin/kin/sdk/migration/MigrationManagerIntegrationTest.kt

```diff
@@ -113,220 +130,290 @@ class MigrationManagerIntegrationTest {
     @LargeTest
     fun start_StartWhileInProcess_MigrationInProcessException() {
         assertFailsWith(MigrationInProcessException::class) {
-            val latch = CountDownLatch(1)
             val migrationManager = getNewMigrationManager(IKinVersionProvider {
                 // waiting so it will be in the middle of the migration process while starting another one.
-                latch.await(timeoutDurationSecondsShort, TimeUnit.SECONDS)
+                Thread.sleep(3000)
                 KinSdkVersion.OLD_KIN_SDK
             })
-            migrationManager.start(migrationManagerListener)
+            migrationManager.start(migrationManagerCallbacks)
             // waiting a bit so the started migration will be in the process.
-            Thread.sleep(2000)
-            migrationManager.start(migrationManagerListener)
-            latch.countDown()
+            Thread.sleep(1000)
+            migrationManager.start(migrationManagerCallbacks)
         }
     }
 
     @Test
     @LargeTest
     fun start_NeedToMigrate_getNewMigratedKinClient() {
```

Comment:
I suggest to move to a more descriptive test name (BDD) style, as we're using kotlin and can put real sentence here with spaces and all

Reply:
what do you mean?
Can you give an example